src/fides/api/alembic/migrations/versions/e2cda8d6abc3_add_dismissed_in_activity_view_to_.py

```diff
@@ -0,0 +1,31 @@
+"""add_dismissed_in_activity_view_to_monitor_task
+
+Revision ID: e2cda8d6abc3
+Revises: 0eef0016cf06
+Create Date: 2025-09-17 23:37:49.678071
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "e2cda8d6abc3"
+down_revision = "0eef0016cf06"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # Add the dismissed_in_activity_view column to monitortask table
+    op.add_column(
+        "monitortask",
+        sa.Column(
+            "dismissed_in_activity_view", sa.Boolean(), nullable=True, default=False
```

Comment:
sorry to make a bit of a nit here and ask you to go back and update this because i know it'll mean you have to make changes downstream - but i'd much prefer we keep this column name more generic, because we very well could (will) end up using this concept of a "dismissed" task in other areas besides just the activity view. the activity view could also easily be renamed "in progress" tomorrow 😄 

while those sort of changes wouldn't _require_ a migration going forward, it'll just make this column feel really stale and it'd be confusing. i'd keep this pretty simple and just say `dismissed`. what do you think?
```suggestion
            "dismissed", sa.Boolean(), nullable=True, default=False
```

Reply:
another nit is that it reads odd to me to have this both be `nullable=True` but to also set its default to `False`. if we're saying it's nullable, then i don't think we should a default value - the null state would indicate an empty/default state.

but really, i actually think this shouldn't be `nullable=True`. i don't think there's a good justification for making the boolean here have 3 states - the task has either been dismissed or it hasn't been dismissed; and the default state should be `False`, i.e. it hasn't been dismissed :)

```suggestion
            "dismissed", sa.Boolean(), nullable=False, default=False
```

Reply:
My mistake, I though t this was applicable to this view only and should have brught this up. If its a generic "dismissed" I can definitely change it but it mill probably have like you said downstream impacts and some other tickets might slip

As far as nullable goes, since this is a new column would the migration take care of adding False to all the existing rows in the table? If we made not nullable=False default=False?